docs/schemas/Cell-Annotation-and-Taxonomy/Aligned_taxonomy_schema.csv

```diff
@@ -0,0 +1,82 @@
+[COLUMN_NAME]_color,0ab06f78-6df2-4555-8990-3eec36b2adbb,,,"Color vector for metadata/taxonomy values in format [COLUMN_NAME]_label. This is ONLY used for molgen-shiny plots, but because of this, some metadata files come with these and some don't and that could cause challenges. Should revisit how to store colors and how to deal with metadata in both formats. Should also agree on a standard for which way is preferred.",,,,,,,,,"assigned metadata, calculated metadata, annotations",
+[COLUMN_NAME]_id,701b32ca-4d95-47c5-8213-e7ab8c5373eb,,,"Same as above, but in this case for the order of metadata values (e.g., the levels of a factor, or ascending order of a numeric)",,,,,,,,,"assigned metadata, calculated metadata, annotations",
+assay,a2ced0f5-4fdb-4ede-b524-2d975a70f21a,,,"In CELLxGENE these correspond to a human-readable modality along with the associated EFO ontology term. We often use the term modality in place of assay (e.g., 'Smart-seq2'corresponds to 'EFO:0008931', '10x 3' v3'corresponds to 'EFO:0009922'). This is called ""library method"" in BKP. Ideally we will agree on a term for this and it will be provided upstream from BICAN. Called Modality in taxonomy Google Sheet.",,,,,,,,,assigned metadata,
+assay_ontology_term_id,12c1b9a3-57fa-44ea-8254-38eb3fa4ba71,,,The ontology ID of the assay term.,,,,,,,,,assigned metadata,
+suspension_type,d6d84f18-90c8-4f4f-849a-fd4d0b665b95,,,"Either ""cell"", ""nucleus"", or ""na"" in CELLxGENE. Called entity in the BKP. We should pick one to use.",,"cell, nucleus, na",value set,,,,,,assigned metadata,
+[batch_condition_columns],5d64eed3-ae83-4b47-864d-cc4b3258e7ce,,,"Zero or more vectors of metadata associated with batches. These are not required, but called out separately by cellxgene for analysis purposes.",,,,,,,,,assigned metadata,
+[addtional uncontrolled metadata],d417074d-5f7a-4bb8-8a7f-11e20fdcca97,,,"Additional uncontrolled cell metadata. These are not required, but any additional columns are allowed by all h5ad formats.",,,,,,,,,"assigned metadata, calculated metadata",
+brain_region,71611b58-f98e-4a69-ad88-c0029a377cbe,region of interest,,"Brain region(s) sampled. Called tissue_ontology_term_id in cellxgene; cell_set structures also defined below; called region_of_interest_label and anatomic_division_label in BKP. Also associated are acronymns, labels, etc.; More generally need to arrive at a way of dealing with brain regions. Note that this slot in the Assigned metadata is meant to deal with cell-level assignments for brain region (e.g., dissection) and NOT cell set summarizations by brain region, which are included below.",,,,,,,,,assigned metadata,
+tissue,33ccbc85-ae35-44d6-8bc6-c2fb91116cf7,tissue type,,"Along with ""tissue"" field, these correspond to UBERON terms for the 'brain region' fields that we have (e.g., 'brain' = 'UBERON_0000955'). In process: we need to discuss how to integrate Allen reference brain atlases for mouse and human.",,,,,,,,,assigned metadata,

```

Comment:
the description of this line is the same as next line (and I believe it just belongs to next line)

Reply:
fixed.